### ISSUE TITLE
Interactive no-drop cursor based on result of handleDragEnd

### ISSUE THREAD

**Issue by PeterKottas** [rcdexta/react-trello#289]
**Is your feature request related to a problem? Please describe.**
It would be great if the result of onDragEnd could change the cursor to no-drop if it returns false.

**Describe the solution you'd like**
 I am pretty sure this is doable with react-dnd. I did something similar before, it might just need some tinkering.

**Describe alternatives you've considered**
No other way to do this in my mind. It has to hook in react-dnd and do the work. It shouldn't be too difficult.

**Comment by dapi** [comment]
Nice! Could you code it?

**Comment by dapi** [comment]
Pay attention to this - https://github.com/rcdexta/react-trello/pull/286

**Comment by PeterKottas** [comment]
Yeah I think so, but I can barely work without TS anymore :D So I'd rather leave this after we clear up #285

**Comment by PeterKottas** [comment]
Yep, #286 is on my radar.
